<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="realizarBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>410</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Realizar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>471</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblTallo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>31</width>
     <height>391</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblRama1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblRama2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblRama3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblRama4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblRama5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl31">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl32">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl33">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl34">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl35">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl39">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl38">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl36">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl37">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl40">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>120</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl49">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl44">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl47">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl43">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl46">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl41">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl42">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl50">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl45">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl48">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl29">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl24">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl27">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl23">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl26">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl21">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl22">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl30">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl25">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl28">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl19">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>280</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl14">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl17">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl13">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl16">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl11">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl12">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl20">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>280</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl15">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>230</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl18">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl9">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl8">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl10">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="historial">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>9</y>
     <width>121</width>
     <height>421</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
